test/prov_config_test.c

```diff
@@ -70,6 +72,38 @@ static int test_recursive_config(void)
     return testresult;
 }
 
+static int test_path_config(void)
+{
+    OSSL_LIB_CTX *ctx = NULL;
+    OSSL_PROVIDER *prov;
+    int testresult = 0;
+    struct stat sbuf;
+
+    if (stat("../test/p_test.so", &sbuf) == -1)
```

Comment:
Is this path really right? I thought the test binary is executed with the test_runs/<test_recipe_name> directory as CWD. The load works because it takes the the directory relative from the module dir which is providers -> `providers/../test/p_test.so` Please check whether you aren't always skipping the test.

Other than this the PR looks good.

Reply:
ok, I've updated to build the full path based on OPENSSL_MODULES, which works for me locally here